app/controllers/orchestrations_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# OrchestrationsController allows to handle orchestrations
+class OrchestrationsController < ApplicationController
+  before_action :can_retry?, only: :retry_bootstrap
+
+  def retry_bootstrap
+    Orchestration.run
+    redirect_to root_path
+  end
+
+  private
+
+  def can_retry?
```

Comment:
The question mark on a method in ruby means that a boolean is expected to be returned. I'd chose another name like `redirect_unless_retryable` 🤓